src/org/kotlin/community/http/benchmarks/vertxweb/VertxWebBenchmark.kt

```diff
@@ -0,0 +1,36 @@
+package org.kotlin.community.http.benchmarks.vertxweb
```

Comment:
Could you please avoid creating folders for each variant? E.g. for ktor netty & jetty it's a single folder.